[2025-08-14T00:44:16.059+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-08-14T00:44:16.099+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: xrs_update_12h.update_swpc_recent scheduled__2025-08-13T00:00:00+00:00 [queued]>
[2025-08-14T00:44:16.106+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: xrs_update_12h.update_swpc_recent scheduled__2025-08-13T00:00:00+00:00 [queued]>
[2025-08-14T00:44:16.107+0000] {taskinstance.py:2306} INFO - Starting attempt 3 of 5
[2025-08-14T00:44:16.120+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): update_swpc_recent> on 2025-08-13 00:00:00+00:00
[2025-08-14T00:44:16.129+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'xrs_update_12h', 'update_swpc_recent', 'scheduled__2025-08-13T00:00:00+00:00', '--job-id', '172', '--raw', '--subdir', 'DAGS_FOLDER/xrs_update_daily.py', '--cfg-path', '/tmp/tmp9sjwmans']
[2025-08-14T00:44:16.131+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=6508) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-08-14T00:44:16.131+0000] {standard_task_runner.py:91} INFO - Job 172: Subtask update_swpc_recent
[2025-08-14T00:44:16.132+0000] {standard_task_runner.py:63} INFO - Started process 6509 to run task
[2025-08-14T00:44:16.190+0000] {task_command.py:426} INFO - Running <TaskInstance: xrs_update_12h.update_swpc_recent scheduled__2025-08-13T00:00:00+00:00 [running]> on host def4e694b617
[2025-08-14T00:44:16.283+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='you' AIRFLOW_CTX_DAG_ID='xrs_update_12h' AIRFLOW_CTX_TASK_ID='update_swpc_recent' AIRFLOW_CTX_EXECUTION_DATE='2025-08-13T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-08-13T00:00:00+00:00'
[2025-08-14T00:44:16.284+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-08-14T00:44:16.297+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-08-14T00:44:16.307+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python /opt/project/src/data/xrs_update.py --mode update --since-minutes 20160 --keep-days 180']
[2025-08-14T00:44:16.320+0000] {subprocess.py:86} INFO - Output:
[2025-08-14T00:44:21.093+0000] {subprocess.py:93} INFO - [INFO] retention applied: >= 2025-02-15 00:44:20.763054+00:00
[2025-08-14T00:44:21.094+0000] {subprocess.py:93} INFO - {"merged_rows_total": 151242, "range": ["2025-05-01 00:00:00+00:00", "2025-08-14 00:41:00+00:00"]}
[2025-08-14T00:44:21.233+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-08-14T00:44:21.236+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-08-14T00:44:21.264+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=xrs_update_12h, task_id=update_swpc_recent, run_id=scheduled__2025-08-13T00:00:00+00:00, execution_date=20250813T000000, start_date=20250814T004416, end_date=20250814T004421
[2025-08-14T00:44:21.324+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-08-14T00:44:21.351+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-08-14T00:44:21.353+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
